### ISSUE TITLE
obj loss hard to converge

### ISSUE THREAD

**Issue by leedewdew** [MohsenZand/ObjectBox#6]
Hi, Thanks for your great work.
When I transplant this label assign to my code, I found obj loss hard to converge.
Will be there many difficult boxes to recognize foreground or background during training due to we treat the target boxes at all scales? 
In other words, whether this method will introduce many fuzzy prediction to learn?

**Comment by MohsenZand** [comment]
Thank you. 
I don't think this is related to  treating the target boxes at all scales, since it is seen in other detectors such as YOLO. 
Individual terms of the loss can be adjusted by  weighting and tweaking the related hyperparameters (such as obj and obj_pw).

**Comment by leedewdew** [comment]
Thank you. I'll try.
